language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSContextOperation.java

```diff
@@ -41,7 +41,7 @@
     TXT_FOLDING_RANGE("text/foldingRange"),
     WS_SYMBOL("workspace/symbol"),
     WS_EXEC_CMD("workspace/executeCommand"),
-    PROJ_MODULES("ballerinaProject/modules"),
```

Comment:
Shall we use the `PROJ_MODULES("ballerinaProject/modules")`? Because even in the new project structure, modules are the components we need to show in the project view.